Fix travis complaining on the 301 redirect

jaybosamiya opened the pull request:
Fix issue mentioned in my comment in #27

jaybosamiya commented:
Whut! That doesn't make sense. We're getting a 301 redirect from the medium post to artplusmarketing, which has a 302 redirect back to the original URL. Running in a browser, I copied the last of the redirects and used it as a URL here, and it still shows an error. Why? :/

jaybosamiya commented:
If it makes sense to someone, please do let me know in a comment here, and I will be happy to fix it. Worst case though, I will just white-list this URL, so that we no longer have the unnecessary Travis complaining going on.

captn3m0 commented:
The first redirect from medium -> artplusmarketing is for the custom domain being used for this publication on medium.

The second redirect is the more curious one:

```
curl https://artplusmarketing.com/10-things-infosec-professionals-need-to-know-about-networking-d159946efc93  -X GET -i
HTTP/2 302 
server: nginx
date: Thu, 17 Aug 2017 14:27:16 GMT
content-type: text/html
content-length: 154
location: https://medium.com/m/global-identity?redirectUrl=https://artplusmarketing.com/10-things-infosec-professionals-need-to-know-about-networking-d159946efc93

<html>
<head><title>302 Found</title></head>
<body bgcolor="white">
<center><h1>302 Found</h1></center>
<hr><center>nginx</center>
</body>
</html>
```

This is by medium to ensure that the user is identified by the cookie set only for medium.com. If you decide to follow the redirect medium does this:

```
HTTP/2 302 
server: nginx
date: Thu, 17 Aug 2017 14:33:44 GMT
content-type: text/html
content-length: 154
location: https://medium.com/m/global-identity?redirectUrl=https://artplusmarketing.com/10-things-infosec-professionals-need-to-know-about-networking-d159946efc93

HTTP/2 302 
date: Thu, 17 Aug 2017 14:33:44 GMT
location: https://artplusmarketing.com/10-things-infosec-professionals-need-to-know-about-networking-d159946efc93?gi=143587189bd2
```

Notice that the final location now includes a `gi` query param. If you follow this redirect as well, you get the final HTML content:

```
HTTP/2 200 
server: nginx
date: Thu, 17 Aug 2017 14:33:45 GMT
content-type: text/html; charset=utf-8
content-length: 242710
```

Haven't checked the travis integration, but increase the --max-redirects parameter (curl) or the equivalent to 5 should fix this?

captn3m0 commented:
awesome_bot has a `--allow-redirect` flag, that might work here?

jaybosamiya commented:
Thanks @captn3m0 ; allowing redirects makes sense, esp since redirects will usually send us to the link we want (even for other links in the future that might change). Added that in, and reverted back to the original link.
